src/python-common/ceph/deployment/service_spec.py

```diff
@@ -137,7 +137,10 @@ def parse(cls, host, require_network=True):
         return host_spec
 
     def validate(self) -> None:
-        assert_valid_host(self.hostname)
+        try:
+            assert_valid_host(self.hostname)
+        except SpecValidationError as e:
+            print(e.args)
```

Comment:
`print`?  Where is this going to print to?  Is this function only ever going to be called from a command line command?